src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  text-transform: uppercase;
+}
+:root{
+  --blue-color:#00ACDC;
+}
+
+h1{
+  max-width: 100%;
+  font-family: 'Roboto', sans-serif;
+  line-height: 100px;
+  text-align: center;
+}
+.container{
+  display: flex;
+  background-color:white
+  height: 60px;
+  width: 200px;
+
+
+  margin: 0;
+}
+a:hover{
+  color:var(--blue-color)
+  text-decoration: underline;
+}
+a {
+ text-decoration: none;
+ color:black;
+ position: relative;
+ line-height: 14px;
+}
+a.is-active::after{
+  color: #var(--blue-color);
+}
+
+
+.laptops{
+  color:#var(--blue-color);
```

Comment:
Similar to the previous comment, the color property is incorrectly using '#var(--blue-color)'. Remove the '#' to correctly use the CSS variable: 'color: var(--blue-color);'.